configuration/pih/htmlforms/covid19Intake.xml

```diff
@@ -349,7 +349,7 @@
                             <span id="calculated-edd-label">
                                 <uimessage code="pihcore.calculatedEstimatedDeliveryDate"/>:&#160;
                             </span><br/>
-                            <span id='calculated-edd' class="calculated-edd value"></span>
+                            <span id='calculated-edd-value' class="calculated-edd value"></span>
```

Comment:
I don't insist on using ids @cioan .  If you are going to write code that simulates using classes, you might as well use them.